lighthouse-core/config/default-config.js

```diff
@@ -94,6 +94,15 @@ const UIStrings = {
   seoCrawlingGroupTitle: 'Crawling and Indexing',
   /* Description of the navigation section within the Search Engine Optimization (SEO) category. Within this section are audits with descriptive titles that highlight ways to make a website accessible to search engine crawlers. */
   seoCrawlingGroupDescription: 'To appear in search results, crawlers need access to your app.',
+  /** Title of the Progressive Web Application (PWA) category of audits. This is displayed at the top of a list of audits focused on topics related to whether or not a site is a progressive web app, e.g. responds offline, uses a service worker, is on https, etc. Also used as a label of a score gauge; try to limit to 20 characters. */
```

Comment:
> Title of the Progressive Web Application (PWA) category of audits

what kind of guidance should a translator have for translating this? We're wandering into the same territory as the metric names... Even if taken literally (do we want them to do that?), the "Progressive" part seems very ambiguous

Reply:
Yeah, I'm not sure.  I think it will end up like FMP and the like.  A full message by itself.  But I don't really know what to give as guidance 😐